src/components/fields/category-select-field.vue

```diff
@@ -132,6 +141,8 @@ export default defineComponent({
     // not sure why it works only using `as`
     const levelValues = ref<CategoryModel[]>(props.values);
 
+    const searchValue = ref<string>('');
```

Comment:
Rename it to `searchQuery`, cause the current naming looks bad.
Also, you don't need to write `ref<string>('')`, TS can easily understand the type based on the default value, so just use `ref('')`